prep-css-box-model/styles.css

```diff
@@ -0,0 +1,13 @@
+body {
+  text-align: center;
+}
+
+button {
+  background-color: red;
+  color: white;
+
+  /* 👇 YOUR CODE HERE 👇 */
+  margin: 20px;
+  padding: 10px;
+  border-style: double 3px darkgray;
```

Comment:
Hi James, try switching up the property of your border styling from `border-style:` to just `border:`